src/kalman.jl

```diff
@@ -124,3 +124,109 @@ function stationary_values(k::Kalman)
     K_inf = A * Sigma_inf * G' * inv(G * Sigma_inf * G' + R)
     return Sigma_inf, K_inf
 end
+
+"""
+computes log-likelihood of period ``t``
+
+##### Arguments
+- `kn::Kalman`: `Kalman` specifying the model. Current values must be the 
+                forecast for period ``t`` observation conditional on ``t-1``
+                observation.
+- `y::AbstractVector`: observations at period ``t``
+##### Returns
```

Comment:
It improves readability to put a new line to separate the end of the inputs and the beginning of the returns.

There are a few other places this shows up in this file.